features/bootstrap/LanguageSelectionContext.php

```diff
@@ -13,7 +13,6 @@ class LanguageSelectionContext extends CentreonContext
     public function theUserWithAutologinEnabled()
     {
         $this->currentPage = new ParametersMyAccountPage($this);
-        $this->currentPage->save();
```

Comment:
Why do you remove the form submit ?

Reply:
This step is supposed to be about the visit of the page, the saving step doesn't make any sense here because the following ones are doing assertions.
This messed up the tests because the assertions were done in a too short of a time frame.

Reply:
indeed, it's the given step. but why is it randomly breaking the TA since 10 days ?